_posts/2020-04-17-gh-actions-lessons-learned.md

```diff
@@ -0,0 +1,34 @@
+# GitHub Actions - Lessons Learned
```

Comment:
Can you add a header like this one (edited for yours, of course, and maybe an `author` tag too):

```md
---
layout: post
title: Forecasting the stock market with pmdarima
subtitle: An end-to-end time series example with python's auto.arima equivalent
tags: [python, time-series, pmdarima, forecasting]
---
```

Reply:
This has been updated. Not sure how that `author` tag works; do I need to add my email or anything?

Reply:
@aaronreidsmith I might need to actually figure that out at the layout level. For now, I think this is g2g and I can go back and amend later if needed